apps/portal/src/app/views/notifications/notifications.component.html

```diff
@@ -56,6 +56,30 @@ <h2 class="p-col-12">Notifications</h2>
         *ngIf="allApplicationsList.length !== 0"
       ></p-dropdown>
     </div>
+    <div class="p-col-12 p-md-3 filter-row">
+      <input
```

Comment:
@LakshayaT 
Check if using this will be more helpful.
https://primeng.org/autocomplete

Reply:
Also can you answer these.
Is the purpose of  this additional filter irrespective of what is loaded?
Isn't it better to have a global search too if needed?

https://www.primefaces.org/primeng-v14-lts/table/filter

Need not be part of this PR. But if we want to have better features maybe that is better?